packages/editor/src/components/post-excerpt/panel.js

```diff
@@ -198,8 +198,7 @@ function PrivateExcerpt() {
 				ref={ setPopoverAnchor }
 				renderToggle={ ( { onToggle } ) => (
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
 						className="editor-post-excerpt__dropdown__trigger"
```

Comment:
I'd remove this `className` since it doesn't seem to be used anywhere: https://wpdirectory.net/search/01J7GMC7M7FDZ5ES9VXDT51Z56